usaspending_api/etl/es_award_template.json

```diff
@@ -1,22 +1,20 @@
 {
-  "index_patterns": null,
   "settings": {
     "index.mapping.ignore_malformed": true,
     "index.max_result_window": null,
     "index.refresh_interval": -1,
     "index": {
       "number_of_shards": 5,
-      "number_of_replicas": 0
+      "number_of_replicas": 0,
+      "sort.field" : ["total_obligation.keyword", "award_id"],
+      "sort.order" : ["desc", "desc"],
+      "sort.missing": ["_last", "_last"]
     },
     "analysis": {
       "analyzer": {
         "stemmer_analyzer": {
           "tokenizer": "standard",
-          "filter": [
-            "standard",
-            "lowercase",
-            "singular_stemmer"
-          ]
+          "filter": ["lowercase", "singular_stemmer"]
```

Comment:
I assume `standard` is no longer a filter in ES 7.1?
Does the tokenizer and filter work well together without overlapping functions?

Reply:
Yeah, the `standard` filter was removed, but according to docs it didn't do anything even in 6.2 -https://www.elastic.co/guide/en/elasticsearch/reference/6.2/analysis-standard-tokenfilter.html I haven't noticed any issues, I assume because it wasn't doing anything in the first place

Reply:
Wow, yeah. According to the page it was last useful in v1.3.

 "Doing nothing since v1.4" is a good band name